packages/xo-server/signin.pug

```diff
@@ -6,46 +6,63 @@ html
     meta(name = 'viewport' content = 'width=device-width, initial-scale=1.0')
     title Xen Orchestra
     meta(name = 'author' content = 'Vates SAS')
-    link(rel = 'stylesheet' href = 'index.css')
+    link(rel = 'stylesheet' href = '/index.css')
```

Comment:
Unfortunately I don't want these kind of changes because it breaks setup with a path prefix, we'll have to find something else…

Reply:
Do you have any ideas?

Reply:
Maybe we should simply use another path, like `/signin-otp`.

Or we could pass a relative path to the signin page, like `../`.